README.md

```diff
@@ -68,52 +78,54 @@ This table provides information about the software prerequisites for the Ansible
 
 | **Ansible Modules** | **OneFS Version** | **Python version** | **Python SDK version** | **Ansible**              |
 |---------------------|-----------------------|--------------------|----------------------------|--------------------------|
-| v3.0.0 | 9.4.x <br> 9.5.x <br> 9.7.x | 3.9 <br> 3.10 <br> 3.11 | 0.3.0.1 | 2.14 <br> 2.15 <br> 2.16 |
+| main | 9.4.x <br> 9.5.x <br> 9.7.x <br> 9.8.x | 3.9 <br> 3.10 <br> 3.11 <br> 3.11 | 0.3.0.1 | 2.14 <br> 2.15 <br> 2.16 <br> 2.16 |
```

Comment:
Python version 3.11 is repeated twice, add 3.12 instead same goes for ansible 2.16. Please fix these.

Reply:
yes, updated